securedrop/tests/test_journalist_api.py

```diff
@@ -20,6 +21,8 @@
 
 from .utils.api_helper import get_api_headers
 
+API_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
Could you import this from models instead of duplicating?

Reply:
I did initially but it makes the test too tautological: "the format I expect is the format the code sends" vs "the format I expect is this." If someone changes the format string in models.py, the test should break.